tests/conftest.py

```diff
@@ -103,3 +105,23 @@ def data_loader(
     return PandasDataLoader(
         example_db_engine, pandas_loader_configuration, table_to_df_convertor
     )
+
+
+@fixture
+def superset_app_ctx():
+    with app.app_context() as ctx:
+        yield ctx
+
+
+@fixture
+def load_sales_dataset():
+    with app.app_context():
+        loader = CsvDatasetLoader(
+            "https://raw.githubusercontent.com/apache-superset/examples-data/lowercase_columns_examples/datasets/examples/sales.csv",
+            parse_dates=["order_date"],
+        )
+        loader.load_table()
+        dataset = loader.load_dataset()
+        yield dataset
+        loader.remove_dataset()
+        loader.remove_table()
```

Comment:
You can reuse the `superset_app_ctx` fixture:

```suggestion
def load_sales_dataset(superset_app_ctx):
    loader = CsvDatasetLoader(
        "https://raw.githubusercontent.com/apache-superset/examples-data/lowercase_columns_examples/datasets/examples/sales.csv",
        parse_dates=["order_date"],
    )
    loader.load_table()
    dataset = loader.load_dataset()
    yield dataset
    loader.remove_dataset()
    loader.remove_table()
```